addons/pager/jquery.tablesorter.pager.js

```diff
@@ -328,7 +328,11 @@
 				if (p.fixedHeight && !p.isDisabled) {
 					h = $.data(table, 'pagerSavedHeight');
 					if (h) {
-						d = h - $b.height();
+						var bs = 0;
+						if($(table).css('border-spacing').split(" ").length>0){
+							bs = $(table).css('border-spacing').split(" ")[0].replace(/[^-\d\.]/g, '');
+						}
+						d = h - $b.height()+(bs*p.size)-bs;
```

Comment:
The demo doesn't appear to work for me... it is because of the `-bs` on the end?